src/CTA.Rules.Config/Utils.cs

```diff
@@ -267,5 +301,50 @@ public static void DownloadFilesToFolder(string s3Bucket, string targetFolder, L
                 }
             });
         }
+
+        public static IEnumerable<string> GetProjectPathsFromSolutionFile(string solutionPath)
+        {
+
+            IEnumerable<string> projectPaths = null;
+            try
+            {
+                SolutionFile solution = SolutionFile.Parse(solutionPath);
+                projectPaths = solution.ProjectsInOrder.Where(p => AcceptedProjectTypes.Contains(p.ProjectType)).Select(p => p.AbsolutePath);
+            }
+            catch (Exception ex)
+            {
+                LogHelper.LogError(ex, $"Error while parsing the solution file {solutionPath}");
+            }
+
+            return projectPaths;
+
+        }
+
+        public static HashSet<SolutionProjectType> AcceptedProjectTypes = new HashSet<SolutionProjectType>()
+        {
+            SolutionProjectType.KnownToBeMSBuildFormat,
+            SolutionProjectType.WebDeploymentProject,
+            SolutionProjectType.WebProject
+        };
+
+        public static bool IsSubPathOf(string subPath, string basePath)
```

Comment:
I feel like these parameters are switched. Isn't the first one the "base" (e.g. C:\something\) and we're checking if the second one is a subpath of that (e.g. C:\something\Project\thing.sln)

Can we not just do something like this?

```
public static bool IsSubPathOf(string basePath, string subPath)
{
    return subPath.StartsWith(basePath);
}
```

It seems to work, but I could be forgetting some edge case https://dotnetfiddle.net/p1X1zK